packages/libdb.rb

```diff
@@ -3,34 +3,22 @@
 class Libdb < Package
   description 'Berkeley DB is a family of embedded key-value database libraries providing scalable high-performance data management services to applications.'
   homepage 'http://www.oracle.com/technetwork/database/database-technologies/berkeleydb/overview/index.html'
-  version '6.2.32'
+  version '5.3.28'
   compatibility 'all'
-  source_url 'https://download.oracle.com/berkeley-db/db-6.2.32.tar.gz'
-  source_sha256 'a9c5e2b004a5777aa03510cfe5cd766a4a3b777713406b02809c17c8e0e7a8fb'
+  source_url 'http://deb.debian.org/debian/pool/main/d/db5.3/db5.3_5.3.28+dfsg1.orig.tar.xz'
```

Comment:
I don't doubt Oracle has bastardized this package.  Does it make sense to change the homepage to https://github.com/berkeleydb/libdb?  We could probably use the release source from here also?

Reply:
Good idea.

Reply:
Done.